modules/ROOT/pages/_partials/push/downloading-configuration-file.adoc

```diff
@@ -40,6 +40,8 @@ Cordova::
 --
 // end::excludeDownstream[]
 `src/mobile-services.json`
+
+NOTE: To run the Cordova app on the Android device you also need to download the `google-services.json` from Firebase and put it on the root folder of your project.
```

Comment:
I think it should be also important to mention that this line:
```
<platform name="android">
  <resource-file src="google-services.json" target="app/google-services.json" />
</platform>
```
should be added to the config.xml 

An alternative could be to add the link to the PhoneGap push plugin documentation: https://github.com/phonegap/phonegap-plugin-push/blob/master/docs/INSTALLATION.md

Reply:
I think this is important info (both from @danielpassos  and @b1zzu ), but I feel that it's landing in the wrong place in the docs. 
BTW, do we rely on phonegap push plugin, or is it just that the aerogear plug in uses same reference?

Reply:
@finp yes we fully relly on phonegap push plugin, the aerogear push plugin is a wrapper around the phonegap one

Reply:
As feedback: the first time I followed the documentation I took quite a lot to figure out that under the neath there was phonegap, and once I discovered it I managed to fix issued like the google-services.json by following the documentation on the phonegap repository

Reply:
@finp Can you point me a better place to put that info?

Reply:
ping @finp